core/base/inc/TSystem.h

```diff
@@ -401,8 +401,8 @@ class TSystem : public TNamed {
    virtual FILE           *OpenPipe(const char *command, const char *mode);
    virtual int             ClosePipe(FILE *pipe);
    virtual TString         GetFromPipe(const char *command);
-   virtual void            Exit(int code, Bool_t mode = kTRUE);
-   virtual void            Abort(int code = 0);
+   virtual void            Exit [[ noreturn ]] (int code, Bool_t mode = kTRUE);
+   virtual void            Abort [[ noreturn ]] (int code = 0);
```

Comment:
I think the suggestion below results in slightly more readable code.  Consider applying the same change in all occurrences below.
```suggestion
   [[noreturn]] virtual void            Exit(int code, Bool_t mode = kTRUE);
   [[noreturn]] virtual void            Abort(int code = 0);
```

Reply:
the problem with this is that the functions then are no longer aligned (the indentation of all function names align perfectly now). But if you prefer like this, I am ok. Another way would be to add as a new line above.

Reply:
This is what I mean:
![image](https://user-images.githubusercontent.com/10653970/194907348-f9d8fe73-c260-432f-939d-9fbc23e7484a.png)